charts/consul/test/terraform/gke/variables.tf

```diff
@@ -19,10 +19,18 @@ variable "init_cli" {
 variable "cluster_count" {
   default     = 1
   description = "The number of Kubernetes clusters to create."
+
+  // We currently cannot support more than 2 cluster
+  // because setting up peering is more complicated if cluster count is
+  // more than two.
+  validation {
+    condition     = var.cluster_count < 3 && var.cluster_count > 0
+    error_message = "The cluster_count value must be 1 or 2."
+  }
```

Comment:
Always a good idea to steal from @ishustava

Reply:
Great artists steal... 😄